Taskfile.yml

```diff
@@ -121,23 +120,23 @@ tasks:
     sources:
       - ./**/*.go
       - go.mod
+    vars:
+      FILE: '{{if eq .OS "Windows_NT"}}.exe{{end}}'
     generates:
-      - "{{.BUILD_DIR}}/{{.NAME}}"
+      - "{{.BUILD_DIR}}/{{.NAME}}{{.FILE}}"
     cmds:
-      - "{{.BUILD_CMD}} {{.CLI_ARGS}} -o {{.BUILD_DIR}}/{{.NAME}}"
+      - "go build {{.CLI_ARGS}} -o {{.BUILD_DIR}}/{{.NAME}}{{.FILE}}"
```

Comment:
This task is used by others, and it needs to build including symbols to work `{{.BUILD_CMD}}` includes `-ldflags="-s"` which removes symbols, at least on windows.